src/main/java/org/minions/devfund/yuri/WhackAMoleApp.java

```diff
@@ -0,0 +1,46 @@
+package org.minions.devfund.yuri;
+
+import java.util.Scanner;
+
+/**
+ * It is in charge to run the WhackAMole game.
+ */
+final class WhackAMoleApp {
+
+    public static final int GRID_DIMENSION = 10;
+    public static final int NUM_ATTEMPTS = 50;
+
+    /**
+     * Constructor Method.
+     */
+    private WhackAMoleApp() { }
+
+    /**
+     * It is in charge to execute the game.
+     * @param args arguments.
+     */
+    public static void main(final String[] args) {
+        Scanner scanner = new Scanner(System.in, "UTF-8");
+        WhackAMole whackAMoleGame = new WhackAMole(NUM_ATTEMPTS, GRID_DIMENSION);
+        boolean givingUp = false;
+        while (!givingUp && !whackAMoleGame.playerWin() && !whackAMoleGame.playerLose()) {
+            whackAMoleGame.printGrid();
+            System.out.println("==================================================");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "==================================================" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)